### ISSUE TITLE
[Request] Steelseries Status XL

### ISSUE THREAD

**Issue by GameGuy1900** [ndeadly/MissionControl#332]
![image](https://user-images.githubusercontent.com/26799797/129474181-31332246-23a2-48b0-9cd5-249de482ca5e.png)
This is the screenshot for that controller.
Thank you very much in advance

**Comment by ndeadly** [comment]
I've whitelisted the controller IDs. See if this is enough to make it work properly

[MissionControl-0.5.0-steelseries-stratus-xl-41bacd7.zip](https://github.com/ndeadly/MissionControl/files/6988391/MissionControl-0.5.0-steelseries-stratus-xl-41bacd7.zip)

**Comment by GameGuy1900** [comment]
No, the controls are all over the place. e.g. left stick up is back.

**Comment by ndeadly** [comment]
Ok, then the report layout must be different to the existing steelseries controllers. Can you use this tool and record the numeric output for each button press? https://github.com/ndeadly/MissionControl/files/6950590/hid_report_viewer_mc.0.5.0.zip

**Comment by celticdude234** [comment]
Any follow up on this controller? I'm operating with the latest version v0.7.0 and my Steelseries Status XL won't even connect. I tried the patch linked above, but that was for version v0.5.0 and my Switch froze on boot (not even an info screen) until I manually removed the patch files. Then I tried the same after downgrading MissionControl to v0.5.0 with the same result. Any help is appreciated!

**Comment by ndeadly** [comment]
No, the person who opened this issue never followed up on my request and nobody else who owns the controller has come along until now.

The zip file above isn't a patch, but a modified build of 0.5.0 that whitelists the controller's vendor and product id in order for it to be recognised. No build of 0.5.0 will work on recent versions of Atmosphere. Here's a new build based on the most recent Mission Control code that should work.

[MissionControl-0.7.0-steelseries-stratus-xl-382c5a1.zip](https://github.com/ndeadly/MissionControl/files/9196654/MissionControl-0.7.0-steelseries-stratus-xl-382c5a1.zip)


You will also probably require an [updated version](https://github.com/ndeadly/MissionControl/files/7628384/hid_report_viewer.zip) of the hid report viewer application I posted above.

**Comment by celticdude234** [comment]
Thanks for the quick response! I'll try that but I'm thinking it's actually a firmware update necessary for the controller itself, which is unnecessarily frustrating. It won't recognize anywhere else that has Bluetooth functionality either and the official software to update the firmware simply doesn't.

If/when I get it functioning elsewhere, I'll follow up with report data for posterity.
